rows/plugins/xls.py

```diff
@@ -132,6 +132,15 @@ def export_to_xls(table, filename_or_fobj, sheet_name='Sheet1', *args,
                 data['style'] = FORMATTING_STYLES[field_type]
             sheet.write(row_index, column_index, value, **data)
 
-    work_book.save(fobj)
-    fobj.flush()
-    return fobj
+    if filename_or_fobj is not None:
+        _, fobj = get_filename_and_fobj(filename_or_fobj, mode='wb')
+        work_book.save(fobj)
+        fobj.flush()
+        return fobj
+    else:
+        output = StringIO.StringIO()
```

Comment:
As the data is binary you may want to use `io.BytesIO`, which is used to binary data (instead of `io.StringIO`, which is used to unicode/text data).